src/Twig/SqlFormatterExtension.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WouterJ\EloquentBundle\Twig;
+
+class SqlFormatterExtension extends \Twig_Extension
```

Comment:
maybe mark this class as `@private`? or do you want to apply the BC policy on that one as well?